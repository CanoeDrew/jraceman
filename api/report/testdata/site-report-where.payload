{
  "name": "site-report-where",
  "where": {
    "site_id": { "op": "eq", "value": "S2" }
  }
}
