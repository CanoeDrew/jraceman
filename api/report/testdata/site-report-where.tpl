{{/*GT: {
  "where": [ "site_id" ]
} */ -}}
This is the test sites report.
{{ $where := where -}}
{{with row (printf "SELECT name, city, phone from site%s" $where.WhereClause) -}}
name={{.name}}, city={{.city}}, phone={{.phone}}
{{- end}}
