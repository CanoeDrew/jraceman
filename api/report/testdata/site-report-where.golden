{
  "HTML": "This is the test sites report.\nname=Awesome Sound, city=Central Coast, phone=888-555-1234\n"
}